ldes-server-compaction/src/main/java/be/vlaanderen/informatievlaanderen/ldes/server/compaction/application/events/FragmentsCompactedEvent.java

```diff
@@ -0,0 +1,13 @@
+package be.vlaanderen.informatievlaanderen.ldes.server.compaction.application.events;
+
+import be.vlaanderen.informatievlaanderen.ldes.server.fragmentation.entities.Fragment;
+
+public class FragmentsCompactedEvent {
```

Comment:
In the current state this event cannot be used without changing the existing code, can you add getters or convert this to a record? Then future listeners won't have to touch the compaction module :)

Reply:
converted to record